packages/teleport/src/Welcome/NewCredentials/NewMfaDevice.tsx

```diff
@@ -132,21 +134,14 @@ export function NewMfaDevice(props: Props) {
             Two-Factor Method
           </Text>
           <Box mb={1}>
-            {mfaOptions.map((opt, index) => {
-              return (
-                <Radio
-                  key={index}
-                  onClick={() => onSetMfaOption(index, validator)}
-                >
-                  <input
-                    type="radio"
-                    checked={mfaType.value === opt.value}
-                    onChange={() => onSetMfaOption(index, validator)}
-                  />
-                  <label>{opt.label}</label>
-                </Radio>
-              );
-            })}
+            <RadioGroup
```

Comment:
I was just comparing the "Mfa Device Optional" story between this branch and master to see if anything has changed. And it looks like this PR improved the form a bit.

All radio buttons are within the same group (because we set the same `name` attribute) which enables navigation through the radio group with arrow keys – that is not present currently on master. Though we focus on another field whenever a radio option is selected so there's not a lot of benefit to it but still, it's a small improvement.